React/Base/RCTUIKit.h

```diff
@@ -385,7 +391,7 @@ CGPathRef UIBezierPathCreateCGPathRef(UIBezierPath *path);
 // An override of an undocumented API that controls the layer's masksToBounds property
 @property (nonatomic) BOOL clipsToBounds;
 @property (nonatomic, copy) NSColor *backgroundColor;
-@property (nonatomic, readwrite, getter=isOpaque) BOOL opaque;
```

Comment:
The UIKit equivalent doesn't have this modifier, so not sure why we did. Removing gets rid of a warning.